packages/components/src/presentation-components/properties/NavigationPropertyTargetSelector.scss

```diff
@@ -1,12 +1,9 @@
-.presentation-navigation-property-select-input {
-  > input,
-  > input:focus-visible {
-    width: 100%;
-    height: 100%;
-    border: none;
-    text-overflow: ellipsis;
-    overflow: hidden;
-    outline: none;
-    padding: 0;
-  }
+.components-editor-container .iui-input-with-icon .iui-select-button.presentation-navigation-property-select-input > input {
```

Comment:
Why all of these additional classes are needed?

Reply:
itwinui CSS selectors override this class, so I added them to increase specificity.

Reply:
Is `.components-editor-container` really needed? If yes, I would suggest adding some class on a `div` in `NavigationPropertyTargetSelector ` so it works property when used outside `component-react` editors.

Reply:
added `.iui-input` class instead of `.components-editor-container`

Reply:
I don't think `.iui-input` is a good solution because that class is added to input element by `@itwin/itwinui-react` and in future versions it might be removed or renamed. I would still suggest adding class on `div` in `NavigationPropertyTargetSelector` component.

Reply:
added new div